nf_core/commands_modules.py

```diff
@@ -219,31 +219,47 @@ def modules_create(
         sys.exit(1)
 
 
-def modules_test(ctx, tool, dir, no_prompts, update, once, profile):
+def modules_test(ctx, tool, dir, no_prompts, update, once, profile, migrate_pytest):
     """
     Run nf-test for a module.
 
     Given the name of a module, runs the nf-test command to test the module and generate snapshots.
     """
     from nf_core.components.components_test import ComponentsTest
 
-    try:
-        module_tester = ComponentsTest(
-            component_type="modules",
-            component_name=tool,
-            directory=dir,
-            no_prompts=no_prompts,
-            update=update,
-            once=once,
-            remote_url=ctx.obj["modules_repo_url"],
-            branch=ctx.obj["modules_repo_branch"],
-            verbose=ctx.obj["verbose"],
-            profile=profile,
+    if migrate_pytest:
+        modules_create(
+            ctx,
+            tool,
+            dir,
+            author="",
+            label="",
+            meta=True,
+            no_meta=False,
+            force=False,
+            conda_name=None,
+            conda_package_version=None,
+            empty_template=False,
+            migrate_pytest=migrate_pytest,
         )
-        module_tester.run()
-    except (UserWarning, LookupError) as e:
-        log.critical(e)
-        sys.exit(1)
+    else:
```

Comment:
should we always test after migrating the pytest?

Reply:
Will the tests work without some fine tuning?